zulipterminal/model.py

```diff
@@ -567,8 +567,14 @@ def update_stream_message(
         if response["result"] == "success":
             old_topic = self.index["messages"][message_id].get("subject", None)
             new_topic = request["topic"]
+            stream_name = self.index["messages"][message_id].get(
+                "display_recipient", None
+            )
+            no_of_messages = len(self.index["edited_messages"])
             if old_topic != new_topic:
-                self.controller.report_success("You changed a message's topic.")
+                self.controller.report_success(
+                    f"You changed {no_of_messages} message's topic from #{stream_name} > {old_topic} to #{stream_name} > {new_topic}."
```

Comment:
I'm not sure if we have a standard way to do this using `report_success`, but this may be clearer if we can provide a highlight style to group the stream+topic in each case.

This may require changes to existing code, so that could be a follow-up.

Reply:
After fixing the format for reporting message upon moving topics, we can look into highlighting as well.